src/server.ts

```diff
@@ -90,40 +88,94 @@ setInterval(() => {
-                }, {
-                    sub: issuerState.did,
-                    iss: issuerState.did,
-                    nbf: issued,
-                    exp: expires,
-                    vc: {
-                        "@context": [
-                            "https://www.w3.org/2018/credentials/v1",
-                            "https://identity.foundation/.well-known/contexts/did-configuration-v0.0.jsonld"],
-                        issuer: issuerState.did,
-                        issuanceDate: new Date(issued * 1000).toISOString(),
-                        expirationDate: new Date(expires * 1000).toISOString(),
-                        type: [
-                            "VerifiableCredential",
-                            "DomainLinkageCredential"
-                        ],
-                        credentialSubject: {
-                            id: issuerState.did,
-                            origin: new URL(issuerState.config.serverBase).origin,
-                        }
-                    }
-                })
+        const tenMinsAgo = new Date().getTime() / 1000 - 10 * 60 as SecondsSinceEpoch
+        const tenMinsFromNow = new Date().getTime() / 1000 + 10 * 60 as SecondsSinceEpoch
+        const issuedSecondsSinceEpoch = Math.round(tenMinsAgo)
+        const expiresSecondsSinceEpoch = Math.round(tenMinsFromNow)
+        const issuedISOString = new Date(issuedSecondsSinceEpoch * 1000).toISOString() as DateTimeString
+        const expiredISOString = new Date(expiresSecondsSinceEpoch * 1000).toISOString() as DateTimeString
+
+        const issuerDid = issuerState.did as LongFormDid
+
+        const linkedDID: DomainLinkageCredentialJWTPayload = {
+            exp: expiresSecondsSinceEpoch,
+            iss: issuerDid,
+            nbf: issuedSecondsSinceEpoch,
+            sub: issuerDid,
+            vc: {
+                "@context": [
+                    "https://www.w3.org/2018/credentials/v1",
+                    "https://identity.foundation/.well-known/did-configuration/v1"
+                ],
+                credentialSubject: {
+                    id: issuerDid,
+                    origin: new URL(issuerState.config.serverBase).origin,
+                },
+                expirationDate: expiredISOString,
+                issuanceDate: issuedISOString,
+                issuer: issuerDid,
+                type: ["VerifiableCredential", "DomainLinkageCredential"]
+            }
+        }
 
-        ]};
+        const linkedDidJws: SignedDomainLinkageCredentialJWTPayload = {
+            unsignedPayload: linkedDID,
+            signedPayload: await issuerState.sk.sign(
+                { kid: issuerState.did + "#signing-key-1"},
+                linkedDID
+            )
+        }
+
+        const response: WellKnownDidConfigurationResponse = {
+            "@context": "https://identity.foundation/.well-known/did-configuration/v1",
+            "linked_dids": [linkedDidJws.signedPayload]
```

Comment:
Similar to above comment, this needs to be snake_cased as it's the object backing the JSON response (which needs to conform to the [Well Known DID Configuration Spec](https://identity.foundation/.well-known/resources/did-configuration/)